sourcetool/pkg/slsa/slsa_types.go

```diff
@@ -24,6 +25,11 @@ const (
 	AllowedOrgPropPrefix                     = "ORG_SOURCE_"
 )
 
+// ControlNames lists the controls managed by sourcetool
+var ControlNames = []ControlName{
```

Comment:
Optional: provide a more distinctive name?  There are other controls that the tool sort of manages (e.g. the various `GH_REQUIRED_CHECK_` types from ghcontrol.CheckNameToControlName based on 'Required Checks').

If you take my other suggestion then I don't think you'll need it, but if you want to keep it, maybe `SlsaL4ControlNames`?  Then it could be used in `slsa.GetRequiredControlsForLevel` so we don't get out of sync?

Reply:
Maybe SourceControlNames ? This is intended to group all the slsa controls handled by the tool in a list , not for a specific level but all of them. We need it to display in `sourcetool status` for example. SlsaControlNames is redundant as it's in the slsa package already.  GH_REQUIRED_CHECK_ are custom controls not part of the spec, right? Or maybe I misunderstood them?

Reply:
How about `AllLevelControls`?